src/envs/behavior_options.py

```diff
@@ -177,11 +177,13 @@ def navigate_to_param_sampler(rng: Generator,
 
 
 def create_navigate_policy(
-    plan: List[Tuple[float, float,
-                     float]], original_orientation: Tuple[float, float, float]
+    plan: List[Tuple[float, float, float]]
 ) -> Callable[[State, "BehaviorEnv"], Tuple[Array, bool]]:
     """Instantiates and returns a navigation option policy given an RRT
     plan."""
+    # We remove the last element from the input policy since this is the
+    # original orientation
```

Comment:
so is the plan in reverse order?

Reply:
maybe clarify what the inputs mean in the docstring? What exactly is an element of `plan`?

Reply:
The plan is in normal, forward order, an each element is a 3-tuple containing `(x, y, rot)` and is interpreted as a waypoint for the robot to pass thru. I needed the original orientation of the robot, which isn't directly accessible from the first element of the plan (since `rot` is only a yaw, and I need roll and pitch as well), so I just tacked it onto the end of the plan.

Reply:
will update docstring to clarigy this